via/static/scripts/video_player/components/VideoPlayerApp.tsx

```diff
@@ -71,24 +72,43 @@ type FilterInputProps = {
 
 function FilterInput({ elementRef, setFilter, filter }: FilterInputProps) {
   return (
-    <Input
-      data-testid="filter-input"
-      aria-label="Transcript filter"
-      classes={classnames(
-        // Match height of search input in sidebar
-        'h-[32px]'
+    <div className="relative">
+      <Input
+        data-testid="filter-input"
+        aria-label="Transcript filter"
+        classes={classnames(
+          // Match height of search input in sidebar
+          'h-[32px]',
+          // Extra padding right to prevent text and "clear" button overlapping
+          'pr-9'
+        )}
+        elementRef={elementRef}
+        onKeyUp={e => {
+          // Allow user to easily remove focus from search input.
+          if (e.key === 'Escape') {
+            (e.target as HTMLElement).blur();
+          }
+        }}
+        onInput={e => setFilter((e.target as HTMLInputElement).value)}
+        placeholder="Search..."
+        value={filter}
+      />
+      {filter && (
+        <Button
+          data-testid="clear-filter-button"
+          title="Clear"
+          onClick={() => setFilter('')}
+          unstyled
+          classes={classnames(
+            'text-grey-7 hover:text-grey-9 font-semibold',
+            'focus-visible-ring',
+            'absolute right-2 top-[6px] w-5 h-5'
+          )}
+        >
+          <CancelIcon className="w-5 h-5" />
+        </Button>
       )}
```

Comment:
I expect we'll want to incorporate some variation of this into the pattern library. I would definitely extract a separate component here though. The `VideoPlayerApp` component is becoming decently large, so we need to keep an eye out for opportunities to extract pieces into their own components.

Reply:
I agree with Rob that it is probably time to extract this as a component (locally).

Reply:
I guess you guys mean the whole "search input"?

Reply:
Yep, what is currently a component local to `VideoPlayerApp`.

Reply:
Gotcha